Failed to Install on Ubuntu 14.04.2 LTS (GNU/Linux 3.13.0-32-generic x86_64)

The error is as the following:

---

root@iZ94z1dayhrZ:~# npm install coffee-script lua-distiller -g
npm http GET https://registry.npmjs.org/coffee-script
npm http GET https://registry.npmjs.org/lua-distiller
npm http 304 https://registry.npmjs.org/lua-distiller
npm http 304 https://registry.npmjs.org/coffee-script
/usr/local/bin/coffee -> /usr/local/lib/node_modules/coffee-script/bin/coffee
/usr/local/bin/cake -> /usr/local/lib/node_modules/coffee-script/bin/cake

> lua-distiller@0.2.3 preinstall /usr/local/lib/node_modules/lua-distiller
> git submodule update -i -r

fatal: Not a git repository (or any of the parent directories): .git
npm WARN This failure might be due to the use of legacy binary "node"
npm WARN For further explanations, please read
/usr/share/doc/nodejs/README.Debian

npm ERR! weird error 128
npm ERR! not ok code 0

---

Please check.

Thank you,

Ying

--- a/README.md
+++ b/README.md
@@ -12,6 +12,11 @@
 npm install coffee-script lua-distiller -g
 ```
 
+[LuaSrcDiet][] 是可选依赖。如果如果需要 minify 功能，
+请确保 `LuaSrcDiet.lua` 在 `$PATH` 中。
+
+[LuaSrcDiet]: https://github.com/LuaDist/luasrcdiet/tree/486129fa1ef1539071d14a366d686f3892c3d43f
+
 ## Usage 用法
 
 Use in command line

--- a/bin/lua-distiller.coffee
+++ b/bin/lua-distiller.coffee
@@ -86,8 +86,6 @@ OUTPUT_PATH_MINIFIED_LUA = ""
 OUTPUT_PATH_MERGED_JIT = ""
 OUTPUT_PATH_MINIFIED_JIT = ""
 
-PATH_TO_LUA_SRC_DIET = path.resolve(__dirname, "../luasrcdiet/")
-#console.log "[lua-distiller::PATH_TO_LUA_SRC_DIET] #{PATH_TO_LUA_SRC_DIET}"
 
 PATH_TO_LUA_JIT = which "luajit"
 #console.log "[lua-distiller::PATH_TO_LUA_JIT] #{PATH_TO_LUA_JIT}"
@@ -226,7 +224,7 @@ fs.writeFileSync OUTPUT_PATH_MERGED_LUA, result
 
 if p.minify
   console.log "minify merged lua file to: #{OUTPUT_PATH_MINIFIED_LUA}"
-  exec "cd #{PATH_TO_LUA_SRC_DIET} && ./LuaSrcDiet.lua #{OUTPUT_PATH_MERGED_LUA} -o #{OUTPUT_PATH_MINIFIED_LUA} "
+  exec "LuaSrcDiet.lua #{OUTPUT_PATH_MERGED_LUA} -o #{OUTPUT_PATH_MINIFIED_LUA} "
 
 if p.luajitify
   console.log "luajit compile merged lua file to #{OUTPUT_PATH_MERGED_JIT}"

--- a/package.json
+++ b/package.json
@@ -11,7 +11,6 @@
     "bin": {
       "lua-distill": "./bin/lua-distiller.coffee"
     },
-    "scripts":{"preinstall": "git submodule update -i -r"},
     "repository": {
       "type": "git",
       "url": "git://github.com/yi/node-lua-distiller.git"

